_posts/2023-02-17-7th-workshop-announcement.md

```diff
@@ -0,0 +1,140 @@
+
+### Who can participate ?
+Anyone working on glaciological modeling using OGGM,
+but also any other global glacier modeler keen on learning more about OGGM! In particular, 
+we would like to encourage model users (current and future) as well as model developers to join 
+us. The number of participants is limited to approximately 30 due to the housing limitations 
+at the University accomodation. Therefore users and developers of OGGM will be given 
+priority in case interest exceeds capacity.
+
+### Location
+The workshop will be held at the [Edinburgh Climate Change Institute (ECCI)](https://edinburghcentre.org/). ECCI is part of the School of GeoSciences, University of Edinburgh. Located in the [heart of the city,](https://goo.gl/maps/Edb3ixpE4D745icW8) is a leading centre for research, teaching, policy and practice for climate action. In general, summers in Edinburgh are reasonably warm and pleasant and the city has a lot to offer including one of the most famouse summer festivals in the world [The Fringe](https://www.edfringe.com/experience/what-is-the-festival-fringe). 
+
+### Transportation
+Edinburgh is the capital city of Scotland (in the UK) therefore, it is very accesible by flight or train. Edinburgh Airport is the closest international airport and Edinburgh Waverley is the name of the central train station. From Amsterdam (NL) there are a regular ferries to Newcastle (England), which is only approx. 2hrs by train from Edinburgh. There are also daily trains from London. 
+
+From the Airport there are several bus routes and tram to the city centre ([Princes street](https://goo.gl/maps/2to6fG8Mgj7DtYQk6)), more information on public transport to and from the Airport can be found [here](https://www.edinburghairport.com/transport-links). Visit the [Lothian buses website](https://www.lothianbuses.com/) for transport information within Edinburgh. 
+
+From the Edinburgh Waverley train station to ECCI there is approx. a 8 min walk distance. And the workshop accommodation is reachable by bus.
+
+
+### Arrival / departure dates
+The workshop starts on the 28th September in the morning and ends on the 1st of September in the afternoon. ??
+
+### Accommodation / meals
+All participants will be housed at the [Pollock Estate halls](https://www.uoecollection.com/summer-stays-at-the-university-of-edinburgh/), located at the foot of Arthur's Seat – the city's iconic ancient volcano. There are single and twin occupancy room options. Accommodation in the Estate includes a full Scottish breakfast. Coffe/Tea sessions and Lunches will be provided via The University of Edinburgh catering bussiness at ECCI. Additionally we will host (one or two?) workshop evening dinners.
+
+### Workshop fee
+The final workshop fee is not confirmed yet, we anticipate approx. £480 for a shared room option and £620 for a single room option. This will include breakfast, workshop fee, excursion and the conference social dinner. 
+
+### Packing list
```

Comment:
Everything below about details can be skipped for now. This should be replaced by a "how to apply" link.